src/uct/ib/rdmacm/rdmacm_md.c

```diff
@@ -45,6 +45,40 @@ ucs_status_t uct_rdmacm_md_query(uct_md_h md, uct_md_attr_t *md_attr)
     return UCS_OK;
 }
 
+int uct_rdmacm_is_addr_resolved(struct rdma_event_channel *event_ch)
+{
+    struct rdma_cm_event *event;
+    int is_resolved, ret;
+
+    /* Fetch an event */
+    ret = rdma_get_cm_event(event_ch, &event);
+    if (ret) {
+        ucs_warn("rdma_get_cm_event() failed: %m");
+        return 0;
+    }
+
+    switch (event->event) {
+    case RDMA_CM_EVENT_ADDR_RESOLVED:
+        is_resolved = 1;
+        break;
+    case RDMA_CM_EVENT_ADDR_ERROR:
+    case RDMA_CM_EVENT_UNREACHABLE:
+        is_resolved = 0;
```

Comment:
maybe add ucs_warn here as it may indicate driver issue. also maybe add event.status to the print in all rdmacm warn parts.

Reply:
no need in here, since this function is intended to check whether the address *can* be resolved